flow2ml/Data_Augumentation.py

```diff
@@ -50,3 +51,31 @@ def applyFlip(self,classPath):
             plt.imsave(classPath+"/FlippedImages/Flipped"+image, cv2.cvtColor(Flipped, cv2.COLOR_RGB2BGR))
           except Exception as e:
             print(f"Flip operation failed due to {e}")
+  
+  def applyRotate(self,classPath):
+    ''' 
+      Applies rotation augmentation to all the images in the given folder. It rotates the images by the given angle (in degrees) in the counter clockwise direction
+      Args : 
+        classPath : (string) directory containing images for a particular class.
+    '''
+    try:
+      os.mkdir(classPath+"/RotatedImages")    
+    except:
+      raise Exception("Unable to create directory for rotated images.")
+
+    for image in list(os.listdir(classPath)):
+      # Read image
+      img = cv2.imread(classPath+"/"+image)
+      if not isinstance(self.operations['rotate'], int):
+        raise Exception("Rotation angle must be an integer.")
```

Comment:
I just want to understand the reason behind throwing except if angle is not in integer datatype. Why can’t we take floor/ceil of number and apply rotation.

Reply:
I had actually put this for checking if it was any other type except for int (like string). But yes, we can round it off instead if it is a decimal. I'll make the changes by rounding it off to the nearest decimal, if that's alright.